### PR TITLE
Link navigation between Edit Schedule and Edit Alarm screens

### PR DESCRIPTION
**Pull request by ChaoticWeg**
Title.

### CODE REVIEW

File: src/models/ScheduleModel.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 import { AlarmModel } from "./AlarmModel";
 
 export interface ScheduleModel {
     alarms: AlarmModel[];
     enabled: boolean;
     key: number;
     name: string;
+
+    // DO NOT SERIALIZE ANYTHING BELOW THIS LINE
+    listItemRef?: ScheduleListItem;
```
Comment (richardkriesman) [inline src/models/ScheduleModel.ts:11]: Assume this is only temporary until we implement actual models, yeah?

Comment (ChaoticWeg) [reply src/models/ScheduleModel.ts:11]: Correct assumption

Comment (ChaoticWeg) [reply src/models/ScheduleModel.ts:11]: It's horrible, awful code that will need to be cleaned up next sprint. Sorry 👎

Comment (richardkriesman) [reply src/models/ScheduleModel.ts:11]: Okay! Just wanted to make sure! ^^

**Review by richardkriesman — APPROVED** [review]

**Pull request merged**